QHyper/optimizers/CEM.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+
+import multiprocessing as mp
+import tqdm 
+from typing import Callable, Any
+
+from .optimizer import HyperparametersOptimizer, Worker, ArgsType, Optimizer
+
+
+class CEM(HyperparametersOptimizer):
+    """Implementation of Cross Entropy Method for hyperparamter tuning
```

Comment:
Rename to all lowercase "CEM.py" -> "cem.py"? 
https://peps.python.org/pep-0008/#package-and-module-names

Reply:
what about QAOA dir?